### ISSUE TITLE
Support for [tabplay~] ?

### ISSUE THREAD

**Issue by jayrope** [enzienaudio/heavy#87]
Will there be support for tabplay~ ? The beauty of this object is, that it allows to play/loop fractions of audio from a table at samplerate. For instance one can fill a second table with a list of indices of positive zero crossings and then have tabplay~ loop between those. Very nice for click free sample exact looping.

**Comment by diplojocus** [comment]
Yep, I think we can support [tabplay~] at some point, need to look at all the features it has. But you should be able to replicate it with a combination of [line~], [hv_vline~], and [phasor~].
